yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -156,87 +185,68 @@ func packAndSendAssessMigrationPayload(status string, errMsg string) {
 		}),
 	}
 
-	unsupportedDatatypesList := lo.Map(assessmentReport.UnsupportedDataTypes, func(datatype utils.TableColumnsDataTypes, _ int) string {
-		return datatype.DataType
-	})
+	explanation, err := buildMigrationComplexityExplanation(source.DBType, assessmentReport, "")
+	if err != nil {
+		log.Errorf("failed to build migration complexity explanation: %v", err)
+	}
+
+	var obfuscatedIssues []callhome.AssessmentIssueCallhome
+	for _, issue := range assessmentReport.Issues {
+		obfuscatedIssue := callhome.AssessmentIssueCallhome{
+			Category:              issue.Category,
+			CategoryDescription:   issue.CategoryDescription,
+			Type:                  issue.Type,
+			Name:                  issue.Name,
+			Description:           issue.Description,
+			Impact:                issue.Impact,
+			ObjectType:            issue.ObjectType,
+			ObjectName:            constants.OBFUSCATE_STRING,
+			SqlStatement:          constants.OBFUSCATE_STRING, // TODO(future): we can obfuscate sensitive info in SQL statement
+			DocsLink:              issue.DocsLink,
+			MinimumVersionFixedIn: issue.MinimumVersionFixedIn,
+		}
 
-	groupedByConstructType := lo.GroupBy(assessmentReport.UnsupportedQueryConstructs, func(q utils.UnsupportedQueryConstruct) string {
-		return q.ConstructTypeName
-	})
-	countByConstructType := lo.MapValues(groupedByConstructType, func(constructs []utils.UnsupportedQueryConstruct, _ string) int {
-		return len(constructs)
-	})
+		// allowing object name for unsupported extension issue type
+		if issue.Type == UNSUPPORTED_EXTENSION_ISSUE_TYPE {
+			// object name(i.e. extension name here) might be qualified with schema so just taking the last part
+			obfuscatedIssue.ObjectName = strings.Split(issue.ObjectName, ".")[len(strings.Split(issue.ObjectName, "."))-1]
+		}
 
-	var unsupportedFeatures []callhome.UnsupportedFeature
-	for _, feature := range assessmentReport.UnsupportedFeatures {
-		if feature.FeatureName == EXTENSION_FEATURE {
-			// For extensions, we need to send the extension name in the feature name
-			// for better categorization in callhome
-			for _, object := range feature.Objects {
-				unsupportedFeatures = append(unsupportedFeatures, callhome.UnsupportedFeature{
-					FeatureName:      fmt.Sprintf("%s - %s", feature.FeatureName, object.ObjectName),
-					ObjectCount:      1,
-					TotalOccurrences: 1,
-				})
-			}
-		} else {
-			var objects []string
-			if slices.Contains(featuresToSendObjectsToCallhome, feature.FeatureName) {
-				objects = lo.Map(feature.Objects, func(o ObjectInfo, _ int) string {
-					return o.ObjectName
-				})
+		for _, sensitiveDescription := range descriptionsIncludingSensitiveInformationToCallhome {
```

Comment:
We had discussed that it does not make sense to send description, right? 🤔  
Do we see any value in sending "description"? It just contains a more verbose explanation of the issue type + sensitive object names. It's more for the user, not helpful when collecting data.

Reply:
no, you remember we discussed that some info in this might be required.
Like column name or object name is not requrid
But object type / datatype like citext etc... can also be there which can be useful.. 
Although this might not be a good way to send that type of info to callhome.